udata_metrics/tasks.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+from udata_metrics.client import InfluxClient
+from udata.models import Dataset
+from udata.tasks import job, task
+from udata.core.metrics.signals import on_site_metrics
+
+from udata_metrics.client import metrics_client_factory
+
+log = logging.getLogger(__name__)
+
+
+@on_site_metrics.connect
+def site_metrics_send(document, **kwargs):
+    write_object_metrics.delay(
+        metrics_client_factory(),
+        document.get_metrics()
+    )
+
+
+@task
+def write_object_metrics(client, metrics):
```

Comment:
We're trying not to pass big payload into tasks signatures. Plus, the metrics could be outdated if the worker is stuck or late (it will send the metrics w/ a wrong timestamp). So initialise the influx client and get the metrics in the task directly. Ideally, do not pass the full document but only its id.